app/assets/javascripts/Components/Modals/submission_url_submit_modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Modal from "react-modal";
+
+class SubmitUrlUploadModal extends React.Component {
+  state = {
+    newUrl: "",
+    newUrlAlias: "",
+  };
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    this.props.onSubmit("https://youtu.be/dQw4w9WgXcQ", "");
+  };
+
+  handleUrlChange = event => {
+    this.setState({newUrl: event.target.value});
+  };
+
+  handleUrlAliasChange = event => {
```

Comment:
What is the goal of including this field as well? I'm not sure I understand what its purpose is.

Reply:
url_alias is the name of the url (if the user wants to give the link a proper name as opposed to just having the name be the link itself). Right now, it is optional and just helps the filename for links be more readable.

Reply:
Ok that makes sense. Let's think about a different label for this one as well, "url alias" to me sounds like they can provide a secondary URL. I had a look at a few ways other software labels this:

- slack uses "Text"
- google docs uses "Text"
- word uses "Text to display"